frontend/css/layout.css

```diff
@@ -64,12 +100,67 @@ li a {
     padding-top: var(--spacing-m);
 }
 
+.search-icons {
+    flex-direction: column;
+    align-items: center;
+}
+
 .ideas {
     text-align: center;
 }
 
 /* Categories section */
 
+.sidebar {
+    position: fixed;
+    list-style-type: none;
+    top: 0;
+    left: 0;
+    height: 100vh;
+    width: 250px;
+    z-index: 1;
+    display: none;
+    backdrop-filter: blur(10px);
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: flex-start;
+    background-color: rgba(255, 255, 255, 0.2);
```

Comment:
is this a contender for a CSS variable?  It makes it more readable what this colour value is and means you can change the variable later on when you choose a different colour - confident you know what effect it will have and confident that you don't need to search through CSS.

Reply:
Background colour of the sidebar has been saved as a css variable in the theming.css file